src/views/agsb.hbs

```diff
@@ -0,0 +1,37 @@
+{{#section "head"}}
+<title>{{{__ "agsb_title"}}} – {{{__ "common_title"}}}</title>
```

Comment:
question: so we don't have an english version?

Reply:
Not at the moment. Bianca just sent me a German version. Shorter texts I usually just translate myself but especially the "Argumente" I don't want to translate freely.

Reply:
@pheeria I will merge it for now and update it once I have an English version.